vars/piperStageWrapper.groovy

```diff
@@ -0,0 +1,109 @@
+import com.sap.piper.Utils
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.ConfigurationLoader
+import groovy.transform.Field
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String STEP_NAME = 'piperStageWrapper'
+
+void call(Map parameters = [:], body) {
+
+    final script = checkScript(this, parameters) ?: this
+    def utils = parameters.juStabUtils ?: new Utils()
+
+    def stageName = parameters.stageName?:env.STAGE_NAME
+
+    // load default & individual configuration
+    Map config = ConfigurationHelper.newInstance(this)
+        .loadStepDefaults()
+        .mixin(ConfigurationLoader.defaultStageConfiguration(this, stageName))
+        .mixinGeneralConfig(script.commonPipelineEnvironment)
+        .mixinStageConfig(script.commonPipelineEnvironment, stageName)
+        .mixin(parameters)
+        .addIfEmpty('stageName', stageName)
+        .dependingOn('stageName').mixin('ordinal')
```

Comment:
Why is the ordinal based on the stage name. Shouldn't it be more a stage configuration?